data/transactions/logic/evalStateful_test.go

```diff
@@ -3254,6 +3254,7 @@ func TestReturnTypes(t *testing.T) {
 
 		"box_create": "int 9; +; box_create",                 // make the size match the 10 in CreateBox
 		"box_put":    "byte 0x010203040506; concat; box_put", // make the 4 byte arg into a 10
+		"bmodexp":    ": byte 0x0102030405; byte 0x010203040506; byte 0x01020304050607; bmodexp",
```

Comment:
Was this required to make the test pass? I wouldn't have expected it to be. This test is really only a sanity check on typing, so if you did this just to exercise the opcode more, I'd rather you do that all in the opcode specific tests.

Reply:
The convention is to only add an element to that table if it's required.  For the majority of opcodes, the test conjures up "reasonable" inputs (like a number for integer inputs or a short string for byte inputs).  This table is for opcodes that need specific inputs in order to function.